datascience_eda/datascience_eda.py

```diff
@@ -90,4 +95,73 @@ def initial_clustering(
 
     # add all plots to result dictionary
 
-    return result
\ No newline at end of file
+    return result
+
+
+def eda_text_columns(df, text_col = None, params = dict()):
+    """Perfroms EDA of text features.
```

Comment:
grammar mistake

Reply:
Thanks for pointing it out! Addressed this in the new [PR](https://github.com/UBC-MDS/datascience_eda/pull/17)